### PR TITLE
Include both creation and last modification dates in gene panels pages

### LINKED ISSUE #4840: Version date for removed gene panel
**Describe the bug**
When a panel is removed, the creation date for the current version is replaced with the date of the latest change (no other new version...), I e the deletion date. Deletion date is useful information, but it would be preferable to keep also the initial version creation date.

**To Reproduce**
1. Go to a panel, note the latest version creation date
2. Click on delete
3. Scroll down to the panel and 
4. See the current deletion date instead of the latest version creation date

**Expected behavior**
Ideally display both dates, at least somewhere. 

**Screenshots**
Screenshot 2024-09-11 at 16 49 03
Screenshot 2024-09-11 at 16 49 19

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Fix #4840 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Deploy to stage, go to gene panels and check the status of `Test Chiara` panel

**Expected outcome**:

Both creating date and deletion date should be show for panel when panel is deleted (hidden)

**Review:**
- [x] code approved by DN
- [x] tests executed by CR, DN

### CODE REVIEW

**Review by dnil — COMMENTED** [review]
Super! Very good with the object id generation time, and showing both. I think the term "created" will look confusing on the panels with many versions. I don't have a super replacement - perhaps "version date"? Also it quickly gets to be many similar numbers. Could we compare the dates, show the modify time only if different perhaps?

File: scout/server/blueprints/panels/templates/panels/panels.html [hunk 1]
```diff
@@ -170,7 +170,7 @@
                 <span class="badge bg-danger" data-bs-toggle="tooltip" title="This panel was removed">Removed</span>
               {% endif %}
             </td>
-            <td>{{ panel.version }} ({{ panel.date.date()}})</td>
+            <td>{{ panel.version }} ({{panel._id.generation_time.strftime('%Y-%m-%d')}} {% if panel.hidden %} deleted:{{ panel.date.date()}} {% endif %})</td>
```
Comment (dnil) [inline scout/server/blueprints/panels/templates/panels/panels.html:173]: 💯

File: scout/server/blueprints/panels/templates/panels/panel.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
       </li>
       <li class="list-group-item">
         <label for="date">Date</label>
-        <span class="float-end" id="date">{{ panel.date.date() }}</span>
+        <span class="float-end" id="date">created:{{panel._id.generation_time.strftime('%Y-%m-%d')}} {% if panel.hidden %} deleted:{{ panel.date.date()}} {% endif %}</span>
```
Comment (dnil) [inline scout/server/blueprints/panels/templates/panels/panel.html:79]: How about dropping "created:" entirely and perhaps adding tooltips with a few words about the date?

Comment (northwestwitch) [reply scout/server/blueprints/panels/templates/panels/panel.html]: I've added version date as you suggested. I think it's self explanatory, no?

**Comment by northwestwitch** [comment]
> Could we compare the dates, show the modify time only if different perhaps? 

Why? What does it add? The deleted will appear just if the panel was deleted, and of course the panel was deleted after it's been created. Also if it's deleted in general is not of much interest anyway, no?

**Comment by dnil** [comment]
It is simple enough that there is no need for a "draft", and it was good enough really early on. We are just commenting on old commits and old comments - it's a control feedback loop lag issue. 😂 

I'll look at the latest version again, but I'm pretty sure that after you added the "if hidden" there is no longer any need to check either dates or show the extra "version date" label for every panel!

File: scout/server/blueprints/panels/templates/panels/panel.html [hunk 3]
```diff
@@ -76,7 +76,7 @@
       </li>
       <li class="list-group-item">
         <label for="date">Date</label>
-        <span class="float-end" id="date">{{ panel.date.date() }}</span>
+        <span class="float-end" id="date">Version date:{{panel._id.generation_time.strftime('%Y-%m-%d')}} {% if panel.hidden %} deleted:{{ panel.date.date()}} {% endif %}</span>
```
Comment (dnil) [inline scout/server/blueprints/panels/templates/panels/panel.html:79]:
```suggestion
        <span class="float-end" id="date">{{panel._id.generation_time.strftime('%Y-%m-%d')}} {% if panel.hidden %} deleted:{{ panel.date.date()}} {% endif %}</span>
```

**Review by dnil — APPROVED** [review]
I would remove the "version date" entirely now that you have a `panel.hidden` check.

**Pull request merged**